src/backend/gl/src/command.rs

```diff
@@ -393,14 +387,39 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
                 }
             }
         }
+
+        if let Some(ref attributes) = self.cache.attributes {
```

Comment:
vertex buffer binding can be done inside and outside a renderpass. The correct place would be before `draw`/`draw_indexed` in my opinion. See the dx12 backend for similar flushing concerning descriptor tables.

Reply:
Good catch, thanks. I will move it to `draw`/`draw_indexed`